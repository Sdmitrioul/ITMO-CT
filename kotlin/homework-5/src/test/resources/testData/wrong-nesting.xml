<mediawiki xml:lang="ru">
    <some>
        <page>
            <title>Простой заголовок</title>
            <another>
                <revision>
                    <timestamp>2020-01-01T00:00:00Z</timestamp>
                    <level>
                        <text bytes="26">Простой текст</text>
                    </level>
                </revision>
            </another>
        </page>
    </some>
</mediawiki>