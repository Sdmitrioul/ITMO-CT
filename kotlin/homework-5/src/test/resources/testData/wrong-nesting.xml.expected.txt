Топ-300 слов в заголовках статей:

Топ-300 слов в статьях:

Распределение статей по размеру:

Распределение статей по времени:
